kuma/search/serializers.py

```diff
@@ -118,12 +117,15 @@ def field_to_native(self, obj, field_name):
 
 
 class DocumentSerializer(BaseDocumentSerializer):
-    excerpt = DocumentExcerptField(source='*')
+    excerpt = serializers.SerializerMethodField('get_excerpt')
     tags = serializers.ChoiceField(read_only=True, source='tags')
     score = serializers.FloatField(read_only=True, source='_meta.score')
     explanation = serializers.SerializerMethodField('get_explanation')
     parent = BaseDocumentSerializer(read_only=True, source='parent')
 
+    def get_excerpt(self, obj):
```

Comment:
Because the code was duplicated it two places, once in the serializer field class and once in `WikiDocumentType.get_excerpt`, which will also return the summary if no highlight.